beacon-chain/blockchain/process_block_helpers.go

```diff
@@ -182,23 +182,23 @@ func (s *Service) sendLightClientFinalityUpdate(ctx context.Context, signed inte
 		}
 	}
 
-	update, err := lightclient.NewLightClientFinalityUpdateFromBeaconState(
+	_, err = lightclient.NewLightClientFinalityUpdateFromBeaconState(
 		ctx,
 		postState,
 		signed,
 		attestedState,
 		attestedBlock,
 		finalizedBlock,
 	)
-
 	if err != nil {
 		return 0, errors.Wrap(err, "could not create light client update")
 	}
 
 	// Return the result
 	result := &ethpbv2.LightClientFinalityUpdateWithVersion{
 		Version: ethpbv2.Version(signed.Version()),
-		Data:    update,
+		// TODO: Get back to this when revisiting events
```

Comment:
I commented out everything event-related. We need to revisit events anyway, so dealing with this now is not that important.